### ISSUE TITLE
Issue reporter: use custom dropdown on Windows

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#56978]
I suggest to use our custom dropdown on Windows for the issue reporter to match the style of dropdowns we have in the workbench:

![image](https://user-images.githubusercontent.com/900690/44451568-6f237c80-a5f4-11e8-94e2-110924eb3e4a.png)

**Comment by cleidigh** [comment]
@RMacfarlane 
If you need anything on this, let me know.
I can also pick this up if you wish.

**Comment by JacksonKearl** [comment]
@bpasero I started looking into this and found that using the custom SelectBox widget would introduce a fairly large amount of code bloat:

In short: the Issue Reporter window doesn't have some of the services needed by SelectBox, particularly `ContextViewService`, which requires a `LayoutService`. While that can be shimmed to work, we're still left without a styling service. Currently the styles are transferred as query params to the new window, and as the `SelectBox` builds upon the List widget, it requires a ton of different color keys, which then need to be serialized and deserialized, and all in all it's just a big can of worms.

At that point I consulted with Rachel and we decided this would be more bloat than its worth,  additionally, I consulted with @eamodio and the current Windows dropdowns seem stylistically and functionally fine. Unless you feel differently I'm going to close as out of scope.
